File: node/fizzBuzzGenerator.js
class FizzBuzzGenerator {
    static generate(input) {
        const isDivisibleBy3 = (value) => value % 3 === 0;
        const isDivisibleBy5 = (value) => value % 5 === 0;

        if(isDivisibleBy3(input) && isDivisibleBy5(input)) {
            return 'FizzBuzz';
        }
        if(isDivisibleBy3(input)) {
            return 'Fizz';
        }
        if(isDivisibleBy5(input)) {
            return 'Buzz';
        }
        return input.toString();
    } 
}

module.exports = FizzBuzzGenerator;

File: python/FizzBuzzGenerator.py
#FizzBuzzGenerator.py
import string
import sys

class FizzBuzzGenerator:

	def validateInput(self,input):
		if type(input) == int:
			if input > 0:
				return True
			else:
				return None
		else:
			return None
	
	def fizzBuzzGenerator(self,input):
		if self.validateInput(input):
			if input%3 == 0 and not(input%5 == 0):
				return "fizz"
			if input%5 == 0 and not(input%3 == 0):
				return "buzz"
			if input%3 == 0 and input%5 == 0:
				return "fizzbuzz"
			if not(input%3 == 0) and not(input%5 == 0):
				return str(input)
		else:
			return None

if __name__ == '__main__':
	main()